php/datasource/class-fieldmanager-datasource-term.php

```diff
@@ -204,9 +204,7 @@ public function presave_alter_values( Fieldmanager_Field $field, $values, $curre
 					$value = sanitize_text_field( substr( $value, 1 ) );
 				}
 
-				// An affordance for our friends at WordPress.com.
-				$term_by = function_exists( 'wpcom_vip_get_term_by' ) ? 'wpcom_vip_get_term_by' : 'get_term_by';
-				$term = call_user_func( $term_by, 'name', $value, $this->taxonomy );
+				$term = get_term_by( 'name', $value, $this->taxonomy );
```

Comment:
Just want to make sure @pjpak gets props for #638, since this change makes that PR obsolete.

Reply:
Noted, thanks!